### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by ShayBox** [Hackzzila/node-ffmpeg-binaries#8]
You should change the url's in your install.js to use always up to date links not static links that never update.
Your ffmpeg is a year old and almost all discord bots using discord.js use voice use your ffmpeg-binaries module.

**Comment by favna** [comment]
https://github.com/Hackzzila/node-ffmpeg-binaries/blob/de39352ace83866f6cb00b04b2e2f5bae9c4dc59/install.js#L14-L19

This was fixed in de39352ace83866f6cb00b04b2e2f5bae9c4dc59 but the issue was never properly closed

**Comment by ShayBox** [comment]
No, I mean always up to date links, not link back to the same repo.
Storing binaries for this means ffmpeg will always be outdated.

**Comment by favna** [comment]
My bad I should've extracted the tarball before drawing conclusions. Now that I see what exactly is inside I fully agree. 

What links would they exactly have to be? I can PR this quite easily but I cannot find a constant link for the 3 binaries (preferably zipped or tarred) that is always the latest. And split by platform, of course.

**Comment by ShayBox** [comment]
```
Git:
    Linux x86: https://johnvansickle.com/ffmpeg/builds/ffmpeg-git-32bit-static.tar.xz
    Linux x64: https://johnvansickle.com/ffmpeg/builds/ffmpeg-git-64bit-static.tar.xz
Release:
    Linux x86: https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-32bit-static.tar.xz
    Linux x64: https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-64bit-static.tar.xz

Windows:
    Static x86: https://ffmpeg.zeranoe.com/builds/win32/static/ffmpeg-latest-win32-static.zip
    Static x64: https://ffmpeg.zeranoe.com/builds/win64/static/ffmpeg-latest-win64-static.zip

    Shared x86: https://ffmpeg.zeranoe.com/builds/win32/shared/ffmpeg-latest-win32-shared.zip
    Shared x64: https://ffmpeg.zeranoe.com/builds/win64/shared/ffmpeg-latest-win64-shared.zip

    Dev x86: https://ffmpeg.zeranoe.com/builds/win32/dev/ffmpeg-latest-win32-dev.zip
    Dev x64: https://ffmpeg.zeranoe.com/builds/win64/dev/ffmpeg-latest-win64-dev.zip

Mac:
    Static x64 https://ffmpeg.zeranoe.com/builds/macos64/static/ffmpeg-latest-macos64-static.zip
    Shared x64: https://ffmpeg.zeranoe.com/builds/macos64/shared/ffmpeg-latest-macos64-shared.zip
    Dev x64: https://ffmpeg.zeranoe.com/builds/macos64/dev/ffmpeg-latest-macos64-dev.zip
```

**Comment by ShayBox** [comment]
All grabbed directly from links from the official website, so they're all supported

**Comment by favna** [comment]
Proposed solution in #9

**Comment by Hackzzila** [comment]
Resolved in 009e4d540aa5bda01cc6781b9f1d1c682260c2ef
